ontotext-yasgui-web-component/src/components/ontotext-yasgui-web-component/ontotext-yasgui-web-component.tsx

```diff
@@ -377,6 +389,17 @@ export class OntotextYasguiWebComponent {
       });
   }
 
+  /**
+   * Aborts the all running count queries if any.
+   */
+  @Method()
+  abortAllQuery(): Promise<any> {
```

Comment:
`abortAllQueries`? The jsdoc says `Aborts the all running count queries` which is confusing. What this method actually does?